websites/B/Bubblez.app/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "DarkMatter",
+    "id": "200612445373464576"
+  },
+  "contributors": [
+    {
+      "name": "embed",
+      "id": "476641014841475084"
+    },
+    {
+      "name": "bob5918",
+      "id": "316673724990488577"
+    }
+  ],

```

Comment:
they didn't contribute to this presence in commits (also it is release so delete them)